xlml/utils/metric.py

```diff
@@ -424,10 +427,16 @@ def get_gce_job_status(
 
   # GCE SSH method
   if not use_startup_script:
-    # check wait status to see if wait_for_ready_queued_resource step is successful
-    wait_task = current_dag.get_task(
-        task_id=f"{benchmark_id}.provision.create_queued_resource.wait_for_ready_queued_resource"
-    )
+    try:
```

Comment:
Could we use if statement to check task exists instead of try catch here? And raise an error if both not the case.

Reply:
Sorry I didn't quite understand. Isn't the line 430 try, except clause serving the purpose? Do you mean a different way to check if `wait_for_ready_queued_resource` exists or not?